webapp/src/js/controllers/contacts.js

```diff
@@ -370,7 +370,7 @@ var _ = require('underscore'),
       $scope.setLeftActionBar(data);
     };
 
-    var getUserHomePlaceSummary = function() {
+    $scope.getUserHomePlaceSummary = function() {
```

Comment:
Using `$scope` makes things less clear because now you have inheritance to deal with. It'll also break when we go to Angular 7+ so we'll have to change this soon anyway.

The easiest way I think is to create a new service for UserHomePlaceSummary so it can be called in both places.

Reply:
Actually the `contacts-content` file already has the facility id [here](https://github.com/medic/medic-webapp/blob/ea79fedaf245042a6c48152f92c9746ce2bae062/webapp/src/js/controllers/contacts-content.js#L50). You shouldn't need to fetch the facility document at all.